include/yaramod/utils/modifying_visitor.h

```diff
@@ -10,6 +10,34 @@
 #include "yaramod/utils/visitor.h"
 
 namespace yaramod {
+/**
+ * This class captures current TokenStream and first and last TokenIi of the expression which it is given
+ * in it's constructor. This data can be very useful at the end of the visit method of ModifyingVisitors
+ * by the cleanUpTokenStreams function.
+ */
+class TokenStreamContext {
+public:
+	TokenStreamContext(Expression* expr)
+		: _oldTS(expr->getTokenStreamSharedPtr())
+		, _oldBeforeFirst(std::prev(expr->getFirstTokenIt()))
+		, _oldAfterLast(std::next(expr->getLastTokenIt()))
+	{
+	}
+	TokenStreamContext(const std::shared_ptr<TokenStream>& oldTS, TokenIt oldBeforeFirst, TokenIt oldAfterLast)
+		: _oldTS(oldTS)
+		, _oldBeforeFirst(oldBeforeFirst)
+		, _oldAfterLast(oldAfterLast)
+	{
+	}
+
+	std::shared_ptr<TokenStream>& oldTS() { return _oldTS; }
```

Comment:
I would prefer `oldTokenStream` instead of `oldTS` since to me it seems like unnecessary abbreviation.

Reply:
Agreed, I fixed it :-)